### PR TITLE
chore: migrate to svelte 5

### PR DESCRIPTION
**Pull request by Michael-Liendo**

### CODE REVIEW

**Comment by Michael-Liendo** [comment]
In a new PR I can migrate Rome Tools to Biome

File: package.json [hunk 1]
```diff
@@ -20,28 +20,25 @@
...
-    "@testing-library/svelte": "^4.0.4",
+    "@sveltejs/adapter-auto": "^3.0.0",
+    "@sveltejs/kit": "^2.5.27",
+    "@sveltejs/package": "^2.3.7",
+    "@sveltejs/vite-plugin-svelte": "^4.0.0",
+    "@testing-library/svelte": "^5.2.6",
     "jsdom": "^22.1.0",
     "publint": "^0.2.2",
-    "svelte": "^4.2.2",
+    "svelte": "^5.0.0",
```
Comment (EstebanBorai) [inline package.json:35]: Can we install the most recent version please?

**Review by LeoBorai — COMMENTED** [review]
WRT Rome, I think we should reevaluate other options.

**Comment by Michael-Liendo** [comment]
Okay!

**Review by LeoBorai — APPROVED** [review]

**Pull request merged**